### PR TITLE
Update importing-pst-files-to-office-365.md

### PR DESCRIPTION
**Pull request by Durga-SC**
Removed South Africa from drive shipping as there is no PST service available

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/importing-pst-files-to-office-365.md [hunk 1]
```diff
@@ -217,7 +217,7 @@ Additionally, to create import jobs in the Microsoft 365 compliance center, one
 
 #### Where is drive shipping available?
 
-Drive shipping is currently available in the United States, Canada, Brazil, the United Kingdom, Europe, India, East Asia, Southeast Asia, Japan, Republic of Korea, Australia, and South Africa. Drive shipping will be available in more regions soon.
+Drive shipping is currently available in the United States, Canada, Brazil, the United Kingdom, Europe, India, East Asia, Southeast Asia, Japan, Republic of Korea, Australia. Drive shipping will be available in more regions soon.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/importing-pst-files-to-office-365.md:220]:
```suggestion
Drive shipping is currently available in the United States, Canada, Brazil, the United Kingdom, Europe, India, East Asia, Southeast Asia, Japan, Republic of Korea, and Australia. Drive shipping will be available in more regions soon.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @Durga-SC, kindly commit the minor suggested change. Many thanks.

**Comment by markjjo** [comment]
@Durga-SC PR merged. Thanks for updating the article.

**Pull request merged**